src/app/public/plugin-resources/code-examples/angular-tree/advanced/angular-tree-demo.module.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  CommonModule
+} from '@angular/common';
+
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  ReactiveFormsModule
+} from '@angular/forms';
+
+import {
+  SkyAngularTreeModule
+} from '@skyux/angular-tree-component';
+
+import {
+  SkyCheckboxModule,
+  SkyRadioModule
+} from '@skyux/forms';
+
+import {
+  SkyFluidGridModule
+} from '@skyux/layout';
+
+import {
+  SkyDropdownModule
+} from '@skyux/popovers';
+
+import {
+  TreeModule
+} from 'angular-tree-component';
+
+import {
+  AngularTreeDemoComponent
+} from './angular-tree-demo.component';
+
+// Build will crash if we try to insert function calls inside the NgModule decorator.
+// To get around this, we just use a variable to refer to the .forRoot() function call.
```

Comment:
"Build will crash" sounds scary and non-explanatory, maybe something like this?

```
// Store the result of the `forRoot` call in a variable to support AoT compilation.
// See: https://github.com/angular/angular/issues/23609
```

Reply:
Done.